src/App.tsx

```diff
@@ -22,14 +22,15 @@ import FilingWarnings from 'pages/Filing/FilingApp/FilingWarnings';
 import UpdateFinancialProfile from 'pages/Filing/UpdateFinancialProfile';
 import ViewUserProfile from 'pages/Filing/ViewUserProfile';
 import type { ReactElement } from 'react';
-import { Suspense, lazy } from 'react';
+import { Suspense, lazy, useEffect } from 'react';
 import {
   BrowserRouter,
   Navigate,
   Outlet,
   Route,
   Routes,
   useLocation,
+  useNavigate,
```

Comment:
```suggestion
```
Also unused.

Reply:
Gonna be a real sad time without VSCode linting...

Reply:
Found a "solution" to the linting issues for now.